lib/SIL/SILVerifier.cpp

```diff
@@ -2682,7 +2682,7 @@ class SILVerifier : public SILVerifierBase<SILVerifier> {
     require(
         !isa<WitnessMethodInst>(branchArg) &&
```

Comment:
Why can't we pass witness methods as BB arguments? Dependencies on opened existentials?